client/src/components/Review Widget/ReviewMetaData.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import './cssFiles/reviewSection.css'
+import ReviewStars from './ReviewStars.jsx'
+
+const ReviewMetaData = (props) => {
+  const [reviewsMeta, setReviewsMeta] = useState({
+    reviewsMeta: [],
+    average: 0
+  })
+
+  useEffect(() => {
+    getReviewsMeta()
+  }, [])
+
+  const getReviewsMeta = () => {
+    axios.get('http://localhost:3000/reviews/meta', {
+      params: {
+        product_id: props.product_id
+      }
+    })
+      .then((data) => {
+        calculateAverage(data.data)
+      })
+      .catch((error) => {
+        console.error(error)
+      })
+  }
+
+  const calculateAverage = (data) => {
+    let currentAverage = 0
+    let reviewCount = 0
+
+    for (const key in data.ratings) {
+      currentAverage = currentAverage + parseInt(key) * parseInt(data.ratings[key])
+      reviewCount = reviewCount + parseInt(data.ratings[key])
+    }
+
+    currentAverage = Math.round((currentAverage / reviewCount) * 10) / 10
+    setReviewsMeta({
+      reviewsMeta: data,
+      average: currentAverage
+    })
+  }
+
+  return (
+    <div className='reviewSummary'>
+      <div className='ratingItem'>{reviewsMeta.average}</div>
+      <ReviewStars className='ratingItem' starRating={reviewsMeta.average} />
```

Comment:
Same thing here as in the other file, not sure about the className being applied here to the ReviewStars call. I would think react would accept this more like a prop and not as a className since ReviewStars is not an element.

Reply:
dang that makes so much sense lol. I couldn't figure out why the classname wasn't appearing in the DOM list in dev tools for review stars but this cleared that up. Thanks for explaining that! Deleted the classname for all the react components where I was trying to apply it in the new branch.